doc/usage/configuration.rst

```diff
@@ -2915,6 +2915,14 @@ Options for the linkcheck builder
 
    .. versionadded:: 4.4
 
+.. confval:: linkcheck_allow_unauthorized
```

Comment:
To prepare for 8.0, perhaps you could say that it will be removed in 8.0?

Reply:
(resolved with slight adjustment: as suggested [here](https://github.com/sphinx-doc/sphinx/pull/11684#issuecomment-1728675463), a flip to opt-out by default in 8.0 would seem to make sense, allowing more time for deployments to adjust)